bin/zkrt

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-zk-related
+zk-related-tags
```

Comment:
I think this might have been renamed at some point? (this script works though, so I assume this is the desired change)